internal/pkg/encryption/keys/tpm2.go

```diff
@@ -53,6 +53,9 @@ func (h *TPMKeyHandler) NewKey(ctx context.Context) (*encryption.Key, token.Toke
 		return nil, nil, err
 	}
 
+	// explicitly clear key from memory since it's not needed anymore
```

Comment:
we seem to be using it below on line 72, so I don't think it's _that_ simple, we still need to pass it down to LUKS

Reply:
ohhh, I missed that completely :sweat_smile: . I saw it in the kms, but missed here